fe1-web/src/features/rollCall/screens/RollCallScanner.tsx

```diff
@@ -178,11 +188,17 @@ const RollCallOpened = () => {
               <PoPTouchableOpacity
                 testID="roll_call_open_add_manually"
                 onPress={() => setInputModalIsVisible(true)}>
-                <Text style={[Typography.base, Typography.accent]}>
+                <Text style={[Typography.base, Typography.accent, Typography.centered]}>
                   {STRINGS.general_enter_manually}
                 </Text>
               </PoPTouchableOpacity>
             </View>
+            <View style={[QrCodeScannerUIElementContainer, styles.counter]}>
+              <Text style={[Typography.base, Typography.accent]}>
+                {/* -1 because we don't count the organizer */}
+                {STRINGS.roll_call_scan_counter}: {attendeePopTokens.current.length - 1}
+              </Text>
+            </View>
```

Comment:
Seeing your screenshot, I think this counter is too similar to the "enter manually" button above it. From a UI/UX point of view, the user might not understand that it's not a button and he might not realize that a counter is there.

I think you should probably think of a better way to include it in the screen. One way would be to just change its style so that is does not resemble that much to the button above. You may also place it differently on the screen, like in a corner. I'll let you figure that out.

Reply:
I think that placing it in the corner is indeed better, but both change are needed. Btw, the android fe also has the counter in the corner.